packages/plugin/package.json

```diff
@@ -41,9 +41,15 @@
   },
   "peerDependencies": {
     "@docusaurus/core": "^3.0.0",
+    "@docusaurus/mdx-loader": "^3.0.0",
     "react": ">=18.0.0",
     "typescript": "^5.0.0"
   },
+  "peerDependenciesMeta": {
+    "@docusaurus/mdx-loader": {
+      "optional": true
```

Comment:
whats this for? probably fine to just be a normap dep since this comes with docusaurus.